source/scripting/Ped.cs

```diff
@@ -26,6 +26,14 @@ public enum HelmetType : uint
 		PilotHeadset = 32768u
 	}
 
+	public enum RagdollType
+	{
+		Normal = 0,
```

Comment:
If you're going to leave out the numbers (something I don't personally like), you don't need to have the first one set to 0 explicitly. It's already 0.